#goal definition
int32 runtime_sec
string topic_name
geometry_msgs/PoseStamped target
---
#result definition
bool success
---
#feedback
float32 remaining_time
geometry_msgs/Pose target
